### DISCUSSION TITLE
modpack/kubejs/1.20.1/KubeJSCourse/KubeJSBasic/Customs/ItemType/MusicDisc

### DISCUSSION THREAD

**Discussion by giscus** [PickAID/CrychicDoc#105] [Announcements]
# modpack/kubejs/1.20.1/KubeJSCourse/KubeJSBasic/Customs/ItemType/MusicDisc

一个包含 Minecraft 开发文档的网站。

https://docs.mihono.cn/zh/modpack/kubejs/1.20.1/KubeJSCourse/KubeJSBasic/Customs/ItemType/MusicDisc

**Comment by ForestMirror** [comment]
您给出的对sounds.json文件的编辑实例中,倒数第二个花括号处是否多了个逗号?
